.template/lib/main.dart

```diff
@@ -65,10 +84,55 @@ class MyHomePage extends StatelessWidget {
             Text(
               FlutterConfig.get('SECRET'),
               style: const TextStyle(color: Colors.black, fontSize: 24),
-            )
+            ),
+            const SizedBox(height: 24),
+            ElevatedButton(
+              onPressed: () => context.go('/page2'),
+              child: const Text("Navigate to Page 2"),
+            ),
           ],
         ),
       ),
     );
   }
 }
+
+class Page3Screen extends StatelessWidget {
+  const Page3Screen({
+    Key? key,
+  }) : super(key: key);
+
+  @override
+  Widget build(BuildContext context) {
+    return Scaffold(
+      appBar: AppBar(
+        title: const Text("Page 3"),
+      ),
+      body: Center(
+          child: ElevatedButton(
+        onPressed: () => context.pop(),
+        child: const Text("Back"),
+      )),
+    );
+  }
+}
```

Comment:
page3 at the end of this class?

Reply:
Fixed in 5a789646d0597aa3c6719c36c9fe667e256ec780